qa/workunits/suites/concurrent_readdir_rename.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -ex
+
+mkdir -p ./dir0/new1
+mkdir -p ./dir0/new2
+touch ./dir0/new1/file{0..9}
+
+count=0
+
+while true ;
+do
+  for i in file0 file1 file2 file3 file4 file5 file6 file7 file8 file9
```

Comment:
```suggestion
  for i in file{0..9}
```

Reply:
That's a valid suggestion, but, this reproducer is basically copied from a (downstream) report 🤷